core/src/main/java/hudson/util/Iterators.java

```diff
@@ -314,12 +315,13 @@ public void remove() {
      * <p>
      * That is, this creates {A,B,C,D} from {A,B},{C,D}.
      */
+    @SafeVarargs
     public static <T> Iterable<T> sequence( final Iterable<? extends T>... iterables ) {
         return new Iterable<T>() {
             public Iterator<T> iterator() {
-                return new FlattenIterator<T,Iterable<? extends T>>(Arrays.asList(iterables)) {
+                return new FlattenIterator<T,Iterable<? extends T>>(ImmutableList.copyOf(iterables)) {
```

Comment:
Not really related but figured I would solve some other compiler warnings while I was here.